s3/container.go

```diff
@@ -301,3 +328,36 @@ func parseMetadata(md map[string]*string) (map[string]interface{}, error) {
 	}
 	return m, nil
 }
+
+func getKmsMasterKeyId(svc *s3.S3, bucketName string) (bucketEncrypted bool, sseAlgortihm string, encryptionKey string) {
+	input := &s3.GetBucketEncryptionInput{
+		Bucket: aws.String(bucketName),
+	}
+
+	output, err := svc.GetBucketEncryption(input)
```

Comment:
[From the docs](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketEncryption.html) `The bucket owner has this permission by default`. In the case that the AWS creds given to stow correspond to an account missing these permissions, it would end up skipping this "set SSE header" step (equivalent to current stow behavior [since SSE is not currently not set])